### PR TITLE
Use default private storage for assets

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by norkans7** [comment]
`BaseAssetStore` in the super class for all the export and those should use the private file storage so they are not accesible to the public on s3

**Comment by nicpottier** [comment]
Can you give us more context as to what is changing here and why?

**Comment by nicpottier** [comment]
Are our current files public then?

**Comment by norkans7** [comment]
`default_storage` uses private as the default ACL  https://github.com/nyaruka/rapidpro/blob/master/temba/flows/views.py#L753

we were using https://github.com/nyaruka/rapidpro/blob/master/temba/utils/s3/s3.py#L12 with public-read so the files are public yes

**Review by nicpottier — APPROVED** [review]

File: temba/assets/models.py [hunk 1]
```diff
@@ -74,29 +73,26 @@ def resolve(self, user, pk):
...
-            # generate our URL manually so that we can force the download name for the user
-            url = public_file_storage.connection.generate_url(
-                public_file_storage.querystring_expire,
-                method="GET",
-                bucket=public_file_storage.bucket.name,
-                key=public_file_storage._encode_name(path),
-                query_auth=public_file_storage.querystring_auth,
-                force_http=not public_file_storage.secure_urls,
-                response_headers={"response-content-disposition": "attachment;filename=%s" % filename},
+        if settings.DEFAULT_FILE_STORAGE == "storages.backends.s3boto3.S3Boto3Storage":  # pragma: needs cover
```
Comment (norkans7) [inline temba/assets/models.py:84]: We are now using boto3

File: pip-freeze.txt [hunk 2]
```diff
@@ -67,7 +67,7 @@ django-mptt==0.8.7
     # via -r pip-requires.txt
 django-redis==4.10.0
     # via -r pip-requires.txt
-django-storages==1.7.1
+django-storages==1.11.1
```
Comment (norkans7) [inline pip-freeze.txt:70]: Update this so we can have a way to pass `http_method` 
https://github.com/nyaruka/rapidpro/pull/3168/files#diff-2c6521d0e448cd239f8e57b29af8b5c9f2cc31d31899b4ccdc593f22d4a4342aR86

**Comment by norkans7** [comment]
Tested all on staging

**Review by rowanseymour — APPROVED** [review]

**Pull request merged**